pkg/build/api/validation/validation.go

```diff
@@ -500,11 +500,14 @@ func validateTrigger(trigger *buildapi.BuildTriggerPolicy, buildFrom *kapi.Objec
 	return allErrs
 }
 
-func validateWebHook(webHook *buildapi.WebHookTrigger, fldPath *field.Path) field.ErrorList {
+func validateWebHook(webHook *buildapi.WebHookTrigger, fldPath *field.Path, allowEnv bool) field.ErrorList {
```

Comment:
@gabemontero that's the "build trigger cause" describer, what you want to look at is:
https://github.com/openshift/origin/blob/master/pkg/cmd/cli/describe/describer.go#L368